analysis_driver/dataset.py

```diff
@@ -316,6 +316,34 @@ def __init__(self, name, most_recent_proc=None, data_threshold=None):
         self._run_elements = None
         self._non_useable_run_elements = None
         self._data_threshold = data_threshold
+        self._species = None
+        self._genome_version = None
+        self._user_sample_id = None
+
```

Comment:
Dataset now has access to species, genome_version, reference_genome and user_sample_id